.github/workflows/release_create_task.yml

```diff
@@ -0,0 +1,54 @@
+name: Create Android App Release Task
+
+on:
+  workflow_dispatch:
+    inputs:
+      app-version:
+        description: 'App Version for Release'
+        required: true
+        default: '5.215.0'
+
+env:
+  ASANA_PAT: ${{ secrets.GH_ASANA_SECRET }}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  create_release_task:
+    name: Create Android App Release Task in Asana
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Release Bridge from Homebrew
+        run: |
+          brew tap cdrussell/aarb
+          brew install aarb
+
+      - name: Create task in Asana
+        run: |
+          AndroidAsanaBridge version=${{ github.event.inputs.app-version }} action=createRelease,tagPendingTasks,addLinksToDescription
```

Comment:
why doesn't it also invoke `removePendingTasks` ?

Reply:
good catch, that needs changing too.